guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+
+## What is liquidity?
+
+Put simply, liquidity on the lightning network is the ability to send or receive bitcoin. In the context of an individual user, liquidity is a measurement of their wallet's ability to send and receive bitcoin over lightning.
+
+### Inbound vs Outbound Liquidity
+
+*Inbound liquidity* is the amount of bitcoin that the user is able to receive over a lightning channel. *Outbound liquidity* is the amount of bitcoin that the user is able to send over a lightning channel.
+
+Receiving bitcoin decreases the user's inbound liquidity and increases their outbound liquidity. On the other hand, sending bitcoin decreases their outbound liquidity and increases their inbound liquidity.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound-mobile@2x.png"
+   alt-text = "Graphic depicting inbound liquidity flowing towards a lightning node and outbound liquidity flowing away from a lightning node"
+   width = 800
+   height = 348
+%}
+
+{% include tip/open.html color="blue" icon="info" label="Sending and receiving" %}
+
+If your app offers the user visibility into the channel management process, you might consider using the term "receive limit" in place of "inbound liquidity" and "send limit" in place of "outbound liquidity".
+
+{% include tip/close.html %}
+
+## Why is liquidity important?
+
+Liquidity directly impacts the user experience. If a user has no outbound liquidity, they can not send bitcoin over lightning. Conversely, if a user has no inbound liquidity, they can not receive bitcoin over lightning.
+
+Sourcing inbound liquidity is very important for onboarding a user to a lightning wallet. Imagine if you were brand new to bitcoin. You download your first wallet, a lightning wallet. Because there is no inbound liquidity, you can not receive any bitcoin. Would you understand what to do next? Would the wallet still seem useful to you?
+
+Helping the user to find inbound liquidity will remove friction from onboarding and make their overall experience with bitcoin easier.
+
+## Identifying different use cases
+
+Before going further, it's important to make a distinction between different use cases for lightning nodes. It is difficult to make firm categories for lightning nodes. However, we can categorize them very generally for our purposes within the Design Guide. Here are some very broad examples of different use cases.
```

Comment:
```suggestion
Before going further, it's important to make a distinction between different use cases for lightning users. It is difficult to make firm categories for lightning user types. However, we can categorize them very generally for our purposes within the Design Guide. Here are some very broad examples of different use cases.
```
The concept of a node hasn't been explained by this point, maybe we need a small section before this explaining that a user is a node (if they are non-custodially using lightning, which they should be) and that these users interact with, and subsequently deal with liquidity differently based on their use case. 

Once we have some content in `Glossay` -> `Node` page (currently working on it) on lightning nodes we can link to that from this section for a deeper dive.

Reply:
For practical reasons, I'd keep this as is and add those links to the nodes page when that page is ready.